02-props-state-component-architecture/props/pokedex/src/pokecard.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import "./Pokecard.css";
+
+export default class Pokecard extends Component {
+	render() {
```

Comment:
I would suggest using a stateless functional component for this since it has no state.